### DISCUSSION TITLE
Artic horizons 2

### DISCUSSION THREAD

**Discussion by Lucas67159** [community/community#121234] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Bonjour,

Avec la dernière version de Kodi 21.0, Artic n'est plus compatible. Je ne peux donc plus utiliser Kodi pour le moment, car il m'est impossible de récupérer ma sauvegarde à cause de l'incompatibilité avec Artic.

**Comment by ebndev** [comment]
Hi @Lucas67159 , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
